query-object/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.iluwatar</groupId>
+        <artifactId>java-design-patterns</artifactId>
+        <version>1.25.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.example</groupId>
+    <artifactId>query-object</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
```

Comment:
I think this extra groupId and artifactId are not needed. You should also include `maven-assembly-plugin` to make the pattern executable from the command line. Please check the other patterns how they have specified their pom.xml.

Reply:
Well I tried to remove it but it failed to pass the build test, I think there shuold be some problems.